RoomMate.Front/RoomMatePortal/src/app/address/address.component.ts

```diff
@@ -129,8 +129,16 @@ export class AddressComponent implements OnInit {
       this.flatAddressService.getAddressByFlatHouseNumb(flatDetailsGetReq).subscribe(response => {
           this.disabledButton = false;
           if (response != null) {
+            this.userExistInList = false;
             this.flatDetails = response;
-          }
+            const loginCurrentUser = JSON.parse(localStorage.getItem("login"));
+            this.flatDetails.loggedUserName = loginCurrentUser;
+            this.flatDetails.users.forEach(element => {
```

Comment:
myślę, że mozną by tu było zrobić prościej -> this.flatDetails.users.Find(x=>x. i twoje warunki)
ale tak też jest poprawnie